### PR TITLE
RRKS Basalt to TitanSMA SM recorder swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing BASALT (2025) from RRKS LC 20
 - Replacing with TitanSMA ~(001719)~ (002343) at same location
 - Using TitanSMA firmware version 4.5.22
 - Updating streams to continuous from `2020-02-09`

### CODE REVIEW

File: install/recorders.csv [hunk 1]
```diff
@@ -1011,7 +1011,7 @@ Kinemetrics,FBA-ES-T-BASALT,BASALT,2024,PTOS,20,7,,0,0,2013-12-16T14:00:00Z,2013
 Kinemetrics,FBA-ES-T-BASALT,BASALT,2024,MLZ,21,60,,0,0,2014-03-25T00:05:00Z,2019-10-31T21:33:00Z
 Kinemetrics,FBA-ES-T-BASALT,BASALT,2024,MLZ,21,0,,0,0,2019-10-31T21:34:00Z,2024-02-15T00:09:00Z
 Kinemetrics,FBA-ES-T-BASALT,BASALT,2025,PTOS,20,7,,0,0,2013-11-18T22:47:00Z,2013-12-16T13:47:29Z
-Kinemetrics,FBA-ES-T-BASALT,BASALT,2025,RRKS,20,94,,0,0,2014-06-06T00:25:25Z,9999-01-01T00:00:00Z
+Kinemetrics,FBA-ES-T-BASALT,BASALT,2025,RRKS,20,94,,0,0,2014-06-06T00:25:25Z,2024-09-25T21:31:00Z
```
Comment (CBurton90) [inline install/recorders.csv:1014]: Seems like the azimuth needs an update to `274`.

https://github.com/GeoNet/tickets/issues/16492#issuecomment-2381634041

Comment (Thomas-Benson) [reply install/recorders.csv:1014]: updated

**Comment by Thomas-Benson** [comment]
TitanSMA installed at a new orientation of `274` degrees

**Review by CBurton90 — APPROVED** [review]

**Review by mnaguit — COMMENTED** [review]
From the ticket, the serial number of the newly-installed TitanSMA is `002343`?
Thanks @Thomas-Benson @CBurton90

**Comment by Thomas-Benson** [comment]
@mnaguit you're correct, it should be `002343`. the SN was wrong in the ticket when I started the PR 🫠

**Review by mnaguit — APPROVED** [review]
Thanks @Thomas-Benson 👍

**Review by CBurton90 — APPROVED** [review]

**Comment by Thomas-Benson** [comment]
Sorry folks, need one last approval after resolving merge conflicts

**Review by mnaguit — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**